owncloudApp/src/main/java/com/owncloud/android/MainApp.kt

```diff
@@ -176,7 +176,7 @@ class MainApp : MultiDexApplication() {
         private const val BETA_VERSION = "beta"
         private const val CLICKS_DEFAULT = 0
 
-        var appContext: Context? = null
+        lateinit var appContext: Context
```

Comment:
Storing the context in a static variable is very ugly

Reply:
Sure, do you have any suggestion? It was included some years ago and probably it will be removed with new architecture. This context is mainly used by operations and managers that we want to refactor.